### PR TITLE
Enforce that only superuser may change verified reports

### PR DESCRIPTION
**Pull request by sliverc**
This protects changing of reports which may already be billed.

### CODE REVIEW

File: timed/tracking/filters.py [hunk 1]
```diff
@@ -122,9 +122,11 @@ def filter_editable(self, queryset, name, value):
 
             # only owner, reviewer or supervisor may change reports
```
Comment (winged) [inline timed/tracking/filters.py]: Maybe should update the comment as well to reflect the implemented policy

File: timed/tracking/filters.py [hunk 2]
```diff
@@ -134,9 +136,11 @@ def filter_editable(self, queryset, name, value):
...
 
             queryset = queryset.exclude(
-                Q(user__supervisors=user) |
-                Q(task__project__reviewers=user) |
-                Q(user=user)
+                (
+                    Q(user__supervisors=user) |
+                    Q(task__project__reviewers=user) |
+                    Q(user=user)
+                ) & Q(verified_by__isnull=True)
```
Comment (winged) [inline timed/tracking/filters.py]: Can we get rid of this code duplication?

File: timed/tracking/filters.py (hunk 1 above)
Comment (sliverc) [reply timed/tracking/filters.py]: adjusted.

File: timed/tracking/filters.py (hunk 2 above)
Comment (sliverc) [reply timed/tracking/filters.py]: Agree. Didn't like this before either. Moved filter query into its own method.

**Pull request merged**